_includes/newsletter.html

```diff
@@ -0,0 +1,23 @@
+<div class="guides-section-white">
+    <div class="row text-center">
+        <div class="col-md-6 col-md-offset-3 newsletter" >
+            <img src="/assets/img/guides/img-newsletter.png" alt="newsletter_image" class="img-center"><br>
+            <label>Newsletter</label>
+            <p>Subscribe to the Gruntwork Newsletter to get notified when updates and new guides are released. No spam. Ever.</p>
+            <form id="newsletter-form" class="col-xs-10 col-xs-offset-1" method="post" action="//gruntwork.us13.list-manage.com/subscribe/post?u=8ccaad0747ac13944cf318e0f&amp;id=786bd365d1">
+                <div class="input-group">
+                    <input type="text" id="email" class="form-control guides-input-box" placeholder="Your Email Address">
```

Comment:
Some things off with the newslatter info, but mainly the vertical alignment of the email address. As I mentioned in other comments though, I would remove the newsletter from this article page and fix these issues on a future PR.


Current: 
<img width="672" alt="Screen Shot 2019-09-05 at 14 11 18" src="https://user-images.githubusercontent.com/1516418/64353017-ec017880-cff4-11e9-8821-3eee68b85be7.png">

Expected:
<img width="517" alt="Screen Shot 2019-09-05 at 15 51 17" src="https://user-images.githubusercontent.com/1516418/64354658-c0cc5880-cff7-11e9-9200-65f5c860f8e4.png">

Reply:
This is a screen resolution issue. It displays properly on mine and other screens below mine. Will fix in my next PR for screen resolutions above mine